src/NitroSharp/InputContext.cs

```diff
@@ -16,7 +16,11 @@ internal enum VirtualKey
         Up,
         Right,
         Down,
-        Skip
+        Skip,
+        Y,
```

Comment:
The idea behind ``VirtualKey`` is that it roughly corresponds to an action rather than a specific physical button. I have no idea what ``Y`` is supposed to do.

Reply:
The problem is that some buttons have different purposes depending on context. I thought it would be better to have `VirtualKey` represent both actions and buttons. For example, `VirtualKey.Backlog` could be mapped to Button Y and Mouse Scroll Up, while `VirtualKey.ButtonY` would be mapped to Button Y and Backslash Key.